src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,51 @@
+package calculator;
+
+public class StringCalculator {
+
+    public static final int ZERO = 0;
+    public static String DELIMITER = ",|:";
+
+    public static int splitAndSum(String text) {
+        if(isNullOrEpmty(text)){
+            return ZERO;
+        }
+        return sum(split(text));
+    }
+
+    private static int sum(String[] values) {
+        int sum = 0;
+        for (String value : values) {
+            sum += toPositiveNumber(value);
+        }
+        return sum;
+    }
+
+    private static boolean isNullOrEpmty(String text) {
+        return text == null || text.isEmpty();
+    }
+
+    private static String[] split(String text) {
+        if(isCustom(text)){
+            return customSplit(text);
+        }
+        return text.split(DELIMITER);
+    }
+
+    private static int toPositiveNumber(String value) {
+        int number = Integer.parseInt(value);
+
+        if(number<0) {
+            throw new RuntimeException("음수는 입력할 수 없습니다.");
+        }
+        return number;
+    }
```

Comment:
```suggestion
    private static String[] split(String text) {
        if(isCustom(text)) {
            return customSplit(text);
        }
        return text.split(DELIMITER);
    }

    private static int toPositiveNumber(String value) {
        int number = Integer.parseInt(value);

        if (number < 0) {
            throw new RuntimeException("음수는 입력할 수 없습니다.");
        }
        return number;
    }
```
코드 컨벤션을 지켜서 작성하시는 연습을 해보는 것도 좋을 것 같습니다.